docs/docs/how-to/cloud/slices-optimization.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Enable Slices API Optimizations"
+description: "Learn how to optimize your Slices on Gatsby Cloud."
+---
+
+> Support for the Gatsby Slice API was added in `gatsby@5.0.0`. Check the status of the Gatsby 5 release in our [Github Discussion](https://github.com/gatsbyjs/gatsby/discussions/36609).
```

Comment:
I'd say the callout to the discussion is not necessary, this will only be visible on the `v5` subdomain I guess

Reply:
I'm not quite sure how our subdomains for docs work... Is it possible we can get these out before beta, but move them to v5 when that subdomain launches?

Reply:
We'd only want to merge this doc when the v5 subdomain is available and the current docs are "frozen" on another branch

Reply:
From Slack discussion: we'll merge this into `v5-docs` and try to get the subdomain set up before beta